packages/language/src/language-server/completion/follow-elements.ts

```diff
@@ -114,43 +147,47 @@ export function getFollowElements(
     case CstNodeKind.WaitStatement_Semicolon:
     case CstNodeKind.WriteStatement_Semicolon:
     case CstNodeKind.DeclareStatement_Semicolon:
-      elements.push({
-        kind: FollowKind.CstNode,
-        types: AllStatementStartKeywordsArray,
-      });
-      break;
+      return [
+        {
+          kind: FollowKind.CstNode,
+          types: AllStatementStartKeywordsArray,
+        },
+      ];
```

Comment:
Minor: I've noticed that we're actually missing the assignment statement follow node here. So a `FollowKind.LocalReference` element needs to be returned in the array as well.